lib/splitclient-rb/engine/matchers/in_list_semver_matcher.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module SplitIoClient
+  class InListSemverMatcher < Matcher
+    MATCHER_TYPE = 'IN_LIST_SEMVER'
+
+    attr_reader :attribute
+
+    def initialize(attribute, list_value, logger, validator)
+      super(logger)
+      @validator = validator
+      @attribute = attribute
+      @semver_list = list_value.map { |item| SplitIoClient::Semver.build(item, logger) }
```

Comment:
you can do the check if `build` returns nil here, eg:
```
def initialize(attribute, list_value, logger, validator)
  super(logger)
  @validator = validator
  @attribute = attribute
  @semver_list = []
  @logger = logger

  list_value.map do |item|
    version = SplitIoClient::Semver.build(item, logger)

    @semver_list << version unless version.nil?
  end
end
```

Reply:
will update